src/artwork.c

```diff
@@ -1744,7 +1744,11 @@ source_item_ownpl_get(struct artwork_ctx *ctx)
 	continue;
 
       ctx->dbmfi->path = dbpli.path;
```

Comment:
This line belongs with the source_item_own_get() call. It is not relevant for the call you have inserted.